src/lib/fetcher.ts

```diff
@@ -27,10 +32,13 @@ export default class Fetcher {
   }
 
   async getContentTypesInChunks (intentList: IntentList): Promise<APIContentType[]> {
-    const ids: string[] = _.uniq(intentList.getIntents().reduce((ids, intent) => {
-      const intentIds = intent.getRelatedContentTypeIds()
-      return ids.concat(intentIds)
-    }, []))
+    const ids: string[] = _.uniq(intentList.getIntents()
+      .filter((intent) => !intent.isEditorInterfaceUpdate())
```

Comment:
Per se this is correct, though I think it makes it harder to understand, as it implies EditInterface updates wouldn't need to know anything about ContentTypes, which isn't correct, right?
Given that the EditorInterface update happens on a ContentType, the ContentType will be fetched either way, so I would rather let the `_.uniq` sort out that duplication.

Reply:
In theory, you don't need to fetch the content type if the only thing you do is updating the editor interface though.
I'd suggest we keep, and add a comment?

Reply:
Ah you're right, the `editContentType` doesn't dispatch an intent.